presets/ragengine/models.py

```diff
@@ -22,7 +22,7 @@ class QueryRequest(BaseModel):
     index_name: str
     query: str
     top_k: int = 10
-    llm_params: Optional[Dict] = None  # Accept a dictionary for parameters
+    model_config = ConfigDict(extra='allow')
```

Comment:
in the recent pr - we have rerank_params map. so now with that we have to change this back to llm_params